samples/dnn/text_detection.py

```diff
@@ -131,16 +173,42 @@ def main():
             for j in range(4):
                 vertices[j][0] *= rW
                 vertices[j][1] *= rH
-            for j in range(4):
-                p1 = (vertices[j][0], vertices[j][1])
-                p2 = (vertices[(j + 1) % 4][0], vertices[(j + 1) % 4][1])
-                cv.line(frame, p1, p2, (0, 255, 0), 1)
+
+
+            vertices = np.asarray(vertices)
+
+            # get cropped image using perspective transform
+            if modelRecognition is not None:
+                cropped = fourPointsTransform(frame, vertices)
+                cropped = cv.cvtColor(cropped, cv.COLOR_BGR2GRAY)
+                cropped = cropped.astype("float32")
+
+                # Create a 4D blob from cropped image
+                blob = cv.dnn.blobFromImage(cropped, size=(100, 32), mean=0.5, scalefactor=2 / 255.0)
```

Comment:
`blobFromImage` subtracts mean values first then applies normalization

Reply:
@dkurt   do you mean I should replace it to 
```
blob = cv.dnn.blobFromImage(cropped, size=(100, 32), mean=127.5, scalefactor=2 / 255.0)
```

Reply:
I guess even
```cpp
                Mat blobCrop = blobFromImage(cropped, 1.0/127.5, Size(), Scalar::all(127.5));
```

Reply:
Yes, that's right